include/rocksdb/secondary_cache.h

```diff
@@ -68,11 +68,14 @@ class SecondaryCache : public Customizable {
   // Lookup the data for the given key in this cache. The create_cb
   // will be used to create the object. The handle returned may not be
   // ready yet, unless wait=true, in which case Lookup() will block until
-  // the handle is ready. is_in_sec_cache is to indicate whether the
-  // handle is possibly erased from the secondary cache after the Lookup.
+  // the handle is ready.
+  // The erase_handle param should be ignored by the implementation
+  // on non-volatile media
```

Comment:
"Non-volatile media" is too much detail for being documented here as a contract of the interface. Perhaps saying that it is a hint from the primary cache that it will be cached there, so the secondary cache is advised to drop it from the cache as an optimization. But it can also be safely ignored.